tests/e2e/test_spyre_cb_scheduler_steps.py

```diff
@@ -174,14 +177,29 @@ def test_prompts_aligned_with_tkv_boundaries(model: str, backend: str,
         max_model_len=max_model_len,
         available_blocks=available_blocks,
         use_cb=True,
+        collect_outputs=check_output,
     )
 
+    if check_output:
+        hf_outputs = generate_hf_output(
+            model=model,
+            prompts=prompts,
+            max_new_tokens=seqs_max_tokens,
+            ignore_eos=True,
+        )
+        compare_results(model=model,
+                        tensor_parallel_size=1,
+                        backend=backend,
+                        vllm_results=cb_outputs,
+                        hf_results=hf_outputs)
+
```

Comment:
wondering if we can somehow not duplicate this whole block multiple times in the test 🤔

Reply:
Yeah.. I had the same thought. A version of this block shows up in a handful of other tests too:

https://github.com/vllm-project/vllm-spyre/blob/main/tests/e2e/test_spyre_basic.py#L89-L98